modules/creating-a-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller with the `Unmanaged` DNS management policy
+
+As a cluster administrator, you can create a custom Ingress Controller with the `Unmanaged` DNS management policy.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* A DNS record is created and published.
+
+.Procedure
+
+. Create a custom resource (CR) file named `sample-ingress.yaml` containing the following:
```

Comment:
Can you just edit the existing,default ingress controller instead of creating a second one?